spec/std/openssl/ssl/sni_spec.cr

```diff
@@ -0,0 +1,130 @@
+require "spec"
+require "socket"
+require "openssl"
+<<-EOC
+To regenerate keys, run:
+openssl genrsa -out server.key 2048
+openssl req -sha256 -new -key server.key -out server.csr -subj '/CN=localhost'
+openssl x509 -req -sha256 -days 3650 -in server.csr -signkey server.key -out server.crt
+rm server.csr
+EOC
+
+# create a server context with private key and certificate chain set
+def new_server_context
+  ctx = OpenSSL::SSL::Context::Server.new
+  ctx.private_key = File.join(__DIR__, "server.key")
+  ctx.certificate_chain = File.join(__DIR__, "server.crt")
+  ctx
+end
+
+# create a client context with CA certificates set to include the default self-signed server certificate
+def new_client_context
+  ctx = OpenSSL::SSL::Context::Client.new
+  ctx.ca_certificates = File.join(__DIR__, "server.crt")
+  ctx
+end
+
+# run an ssl server with the supplied server context and port
+def run_server(q, port, tls = nil)
+  q.send "start"
+  tcpServer = TCPServer.new port
```

Comment:
please, use `snake_case` (ditto all below) instead of `camelCase`.

Reply:
Thanks. I reread the style guide again. Don't know where I got this idea from.